streams/src/main/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyWindowStore.java

```diff
@@ -68,10 +65,9 @@ public V fetch(final K key, final long time) {
     }
 
     @Override
-    @Deprecated
     public WindowStoreIterator<V> fetch(final K key,
-                                        final long timeFrom,
-                                        final long timeTo) {
+                                        final Instant timeFrom,
```

Comment:
Seems we are removing the `ApiUtils.validateMillisecondInstant` check here, is that intentional?

Reply:
Ditto below.

Reply:
Yes, default implementation is doing this convertion already.